src/tlo/methods/cervical_cancer.py

```diff
@@ -0,0 +1,1542 @@
+
+        rate_diagnosed_cc = n_diagnosed_past_year / n_women_alive
+
+        n_women_living_with_diagnosed_cc = \
+            (df['ce_date_diagnosis'].notnull()).sum()
+
+        n_women_living_with_diagnosed_cc_age_lt_30 = \
+            (df['ce_date_diagnosis'].notnull() & (df['age_years'] < 30)).sum()
+        n_women_living_with_diagnosed_cc_age_3050 = \
+            (df['ce_date_diagnosis'].notnull() & (df['age_years'] > 29) & (df['age_years'] < 50)).sum()
+        n_women_living_with_diagnosed_cc_age_gt_50 = \
+            (df['ce_date_diagnosis'].notnull() & (df['age_years'] > 49)).sum()
+
+        out.update({"rounded_decimal_year": rounded_decimal_year})
+        out.update({"n_deaths_past_year": n_deaths_past_year})
+        out.update({"n_treated_past_year": n_treated_past_year})
+        out.update({"prop_cc_hiv": prop_cc_hiv})
+        out.update({"n_diagnosed_past_year_stage1": n_diagnosed_past_year_stage1})
+        out.update({"n_diagnosed_past_year_stage2a": n_diagnosed_past_year_stage2a})
+        out.update({"n_diagnosed_past_year_stage2b": n_diagnosed_past_year_stage2b})
+        out.update({"n_diagnosed_past_year_stage3": n_diagnosed_past_year_stage3})
+        out.update({"n_diagnosed_past_year_stage4": n_diagnosed_past_year_stage4})
+        out.update({"n_ever_diagnosed": n_ever_diagnosed})
+        out.update({"n_screened_xpert_this_month": n_screened_xpert_this_month})
+        out.update({"n_screened_via_this_month": n_screened_via_this_month})
+        out.update({"n_women_alive": n_women_alive})
+        out.update({"n_ever_screened": n_ever_screened})
+        out.update({"n_women_vaccinated": n_women_vaccinated})
+        out.update({"n_vaginal_bleeding_stage1": n_vaginal_bleeding_stage1})
+        out.update({"n_vaginal_bleeding_stage2a": n_vaginal_bleeding_stage2a})
+        out.update({"n_vaginal_bleeding_stage2b": n_vaginal_bleeding_stage2b})
+        out.update({"n_vaginal_bleeding_stage3": n_vaginal_bleeding_stage3})
+        out.update({"n_vaginal_bleeding_stage4": n_vaginal_bleeding_stage4})
+        out.update({"n_diagnosed_past_year": n_diagnosed_past_year})
+        out.update({"n_women_alive": n_women_alive})
+        out.update({"rate_diagnosed_cc": rate_diagnosed_cc})
+        out.update({"cc": cc})
+        out.update({"n_women_living_with_diagnosed_cc": n_women_living_with_diagnosed_cc })
+        out.update({"n_women_living_with_diagnosed_cc_age_lt_30": n_women_living_with_diagnosed_cc_age_lt_30})
+        out.update({"n_women_living_with_diagnosed_cc_age_3050": n_women_living_with_diagnosed_cc_age_3050})
+        out.update({"n_women_living_with_diagnosed_cc_age_gt_50": n_women_living_with_diagnosed_cc_age_gt_50})
+        out.update({"n_diagnosed_1_year_ago": n_diagnosed_1_year_ago})
+        out.update({"n_diagnosed_1_year_ago_died": n_diagnosed_1_year_ago_died})
+
+        print(self.sim.date, 'total_none:', out['total_none'], 'total_hpv:', out['total_hpv'], 'total_cin1:',out['total_cin1'],
```

Comment:
move to using the logger:
i.e. logger.info(key='cervical_cancer_stats_every_month', description='XX', data=out)